exchanges/kraken/kraken_types.go

```diff
@@ -718,7 +708,7 @@ type WsCancelOrderRequest struct {
 type WsCancelOrderResponse struct {
 	Event        string `json:"event"`
 	Status       string `json:"status"`
-	ErrorMessage string `json:"errorMessage"`
+	ErrorMessage string `json:"errorMessage,omitempty"`
```

Comment:
This change doesn't seem necessary. The code for
`if msg, err = jsonparser.GetString(resp, "errorMessage"); err != nil {` is reading from bytes and can figure out if its missing and return. Otherwise, can you tell me how its used since I missed it?

Reply:
I'm using standard json.Marshal with `WsCancelOrderResponse` in the mock handler.
This is just to make the mock responses match what the live endpoint gives :)